src/audit-log-api/src/handlers/createAuditLog.test.ts

```diff
@@ -1,14 +1,18 @@
+jest.mock("src/use-cases/validateCreateAuditLog")
```

Comment:
Nit: This feels like an odd place to put a mock? I'd normally expect imports to be the first thing in a file?

Reply:
We had an issue with mocking modules when we used `jest.mock()` after imports previously. However, I put it after imports in this test and it seems to be okay. 🤷‍♂️

Updated.